media/js/src/views/self-talk-road.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+], function($, _, Backbone) {
+    /**
+     * A view for adding avatar on the self-talk road that changes
+     * as the user selects form elements.
+     */
+    var SelfTalkRoad = Backbone.View.extend({
+        updateCanvasDimensions: function() {
+            var $container = $('.worth-self-talk-road:first');
+            var $canvas = $(this.canvas);
+            $canvas.attr('width', $container.width());
+            $canvas.attr('height', $container.height());
+        },
+        initialize: function() {
+            var $blocks = $('.worth-statement-block,.worth-refutation-block');
+            if ($blocks.length < 1) {
```

Comment:
Because I don't want to run this code on pages that don't contain this block. I'm not putting this code in a separate file specified by `js_template_file` because I'd like it to be part of the main require.js app.